.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,17 @@
+name: Feature request
```

Comment:
This was ~~completely copied~~ heavily inspired from k6 repo 🙂

Any additional fields we'd benefit from having?